src/features/party/PartiesProvider.tsx

```diff
@@ -1,24 +1,34 @@
 import React, { useEffect, useState } from 'react';
 import type { PropsWithChildren } from 'react';
 
-import { useMutation, useQuery } from '@tanstack/react-query';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
 
-import { useAppMutations, useAppQueries } from 'src/core/contexts/AppQueriesProvider';
+import { useAppQueries } from 'src/core/contexts/AppQueriesProvider';
 import { createContext } from 'src/core/contexts/context';
 import { delayedContext } from 'src/core/contexts/delayedContext';
 import { createQueryContext } from 'src/core/contexts/queryContext';
 import { DisplayError } from 'src/core/errorHandling/DisplayError';
 import { Loader } from 'src/core/loading/Loader';
+import { useLaxInstanceData } from 'src/features/instance/InstanceContext';
 import { NoValidPartiesError } from 'src/features/instantiate/containers/NoValidPartiesError';
-import { useShouldFetchProfile } from 'src/features/profile/ProfileProvider';
-import type { IParty } from 'src/types/shared';
-import type { HttpClientError } from 'src/utils/network/sharedNetworking';
+import { useProfile, useShouldFetchProfile } from 'src/features/profile/ProfileProvider';
+import { fetchAllParties } from 'src/queries/queries';
+import { getCookieValue, setCookie } from 'src/utils/cookieUtils';
+import type { IInstance, IInstanceOwner, IParty } from 'src/types/shared';
+
+export const altinnPartyIdCookieName = 'AltinnPartyId';
```

Comment:
For some reason the name of this cookie is configurable in backend. Not sure why.